plugins/gstperf.h

```diff
@@ -36,6 +36,13 @@ G_BEGIN_DECLS
 typedef struct _GstPerf GstPerf;
 typedef struct _GstPerfClass GstPerfClass;
 
+struct _GstPerfClass
+{
+  GstBaseTransformClass parent_class;
+  
+  void (*on_bitrate) (GstPerf * self, gdouble bitrate);
```

Comment:
This is only needed if you want to designate a default handler for this signal. In your case, you have an invalid handler associated.